frontend/src/scenes/pipeline/pipelineBatchExportConfigurationLogic.tsx

```diff
@@ -0,0 +1,121 @@
+import { afterMount, connect, kea, key, listeners, path, props, reducers, selectors } from 'kea'
+import { forms } from 'kea-forms'
+import { loaders } from 'kea-loaders'
+import { router } from 'kea-router'
+import api from 'lib/api'
+import { teamLogic } from 'scenes/teamLogic'
+import { urls } from 'scenes/urls'
+
+import { BatchExportConfiguration, BatchExportService, PipelineNodeTab, PipelineStage } from '~/types'
+
+import { pipelineDestinationsLogic } from './destinationsLogic'
+import type { pipelineBatchExportConfigurationLogicType } from './pipelineBatchExportConfigurationLogicType'
+
+export interface PipelineBatchExportConfigurationLogicProps {
+    service: BatchExportService['type'] | null
+    id: string | null
+}
+
+// TODO:
```

Comment:
Ignore this file - it doesn't yet work & that's ok, I'll update it after https://github.com/PostHog/posthog/pull/21767 is shipped in a separate PR